### DISCUSSION TITLE
Consider adding RegEx documentation to the official Scoop Wiki?

### DISCUSSION THREAD

**Discussion by anderlli0053** [ScoopInstaller/Scoop#5227] [Ideas]
It would be good to have docs, or at least some examples on how to properly use RegEx in Scoop, because searching for `Using regex in scoop manifests` on Google, does not return anything useful.

**Comment by niheaven** [comment]
Please search `Using regex in PowerShell`, or ref: https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_regular_expressions?view=powershell-7.2

**Reply by anderlli0053** [reply]
Ok, i will, thank you.
